content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -15,8 +15,7 @@ is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attrac
 a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (either as a preference or a
 hard requirement). _Taints_ are the opposite -- they allow a node to repel a set of pods.
 
-_Tolerations_ are applied to pods, and allow (but do not require) the pods to schedule
-onto nodes with matching taints.
+_Tolerations_ are applied to pods, if it matches the Taints in any node it is scheduled, but it does not guarantee as scheduler has other paramters to decide.
```

Comment:
Building on @divya-mohan0209's proposed language. 

@Huang-Wei Does this answer your concerns about the cause-and-effect relationship of Tolerations and the scheduler?
```suggestion
_Tolerations_ are applied to pods. Tolerations allow the scheduler to schedule pods with matching taints that match the scheduler's other requirements.
```

Reply:
I agree with @zacharysarah and it sounds way better!

Reply:
Just one doubt aren't we removing that explicit exception?

Reply:
@dipankardas011 Can you say which exception specifically? I'm looking at your original changes and can't immediately see what you mean.

Reply:
It is not explicitly saying that it may not be scheduled like in the original content where it gives a hint. As it didn't sound correct that's why this PR🙂

Reply:
if its okay then I can add this suggestion

Reply:
@dipankardas011 Please do add the suggestion.

Reply:
```suggestion
_Tolerations_ are applied to pods. Tolerations allow the scheduler to schedule pods with matching taints that meet the scheduler's other requirements. even though it's not guaranteed!
```

Is it okay?

Reply:
@dipankardas011 Thanks, I see what you're going for now. I think one more set of edits can do it. 👍🏻

Reply:
@dipankardas011 Does this work? Adding a link to pod priority and preemption may help others understand the larger context.

If you're okay with this suggestion, go ahead and commit it. We're close!

```suggestion
_Tolerations_ are applied to pods. Tolerations allow the scheduler to schedule pods with matching taints. Tolerations allow scheduling but don't guarantee scheduling: the scheduler also [evaluates other parameters](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/) as part of its function.
```

Reply:
That would be a good addition, Thanks👍

Reply:
So committing this change.

Reply:
@dipankardas011 Yes please.